test/dynamo/test_graph_break_messages.py

```diff
@@ -809,7 +881,10 @@ def fn(x):
 
 from user code:
    File "test_graph_break_messages.py", line N, in fn
-    if x.sum() > 0:""",
+    if x.sum() > 0:
+
+Set TORCHDYNAMO_VERBOSE=1 for the internal stack trace. For even more developer context, set TORCH_LOGS="+dynamo"
```

Comment:
is there a way to chop this off, it's kind of annoying to read this in all of the expecttests